From 6959dc6aadbe77edc3d0915ae006848309c20662 Mon Sep 17 00:00:00 2001
From: Alexey Tikhonov <atikhono@redhat.com>
Date: Thu, 7 Dec 2023 09:47:25 +0100
Subject: [PATCH] DP: reduce log level in case a responder asks for unknown
 domain

Since 9358a74d3a56c738890353aaf6bc956bfe72df99 a domain might be
skipped by 'ad_enabled_domains' option

Reviewed-by: Sumit Bose <sbose@redhat.com>
(cherry picked from commit 39cd0baa06742b349ed763aa40ea4de366e80f1a)
---
 src/providers/data_provider/dp_request.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/providers/data_provider/dp_request.c b/src/providers/data_provider/dp_request.c
index df2db2666..9c0fcf1f5 100644
--- a/src/providers/data_provider/dp_request.c
+++ b/src/providers/data_provider/dp_request.c
@@ -204,7 +204,8 @@ dp_req_new(TALLOC_CTX *mem_ctx,
     if (domainname != NULL) {
         dp_req->domain = find_domain_by_name(be_ctx->domain, domainname, true);
         if (dp_req->domain == NULL) {
-            DEBUG(SSSDBG_CRIT_FAILURE, "Unknown domain: %s\n", domainname);
+            /* domain might be skipped by 'ad_enabled_domains' option */
+            DEBUG(SSSDBG_CONF_SETTINGS, "Unknown domain: %s\n", domainname);
             return ERR_DOMAIN_NOT_FOUND;
         }
     }
-- 
2.41.0

